### PR TITLE
style(rxapp): 1.x FRMW-1310 rxApp Background Color

### PR DESCRIPTION
**Pull request by ssalinas24**
JIRA: https://jira.rax.io/browse/FRMW-1310

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] Design LGTM

### CODE REVIEW

**Comment by ssalinas24** [comment]
## Before
before

## After
after_1x

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org/
```
Comment (CITguy) [inline .npmrc]: 👍

**Comment by cathysiller** [comment]
Design LGTM

**Comment by ElementE** [comment]
Design LGTM

**Pull request merged**